model-view-viewmodel/README.md

```diff
@@ -0,0 +1,227 @@
+---
+layout: pattern
+title: Model-View-ViewModel
+folder: model-view-viewmodel
+permalink: /patterns/model-view-viewmodel/
+categories: Architectural
+tags:
+ - Decoupling
+---
+
+## Also known as
+
+model–view–binder
+
+## Intent
+
+To apply "Separation of Concerns" to separate the logic from the UI components and allow developers to work on UI without affecting the logic and vice versa.
```

Comment:
You can link to https://java-design-patterns.com/principles/#separation-of-concerns

Reply:
done